src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/CommitmentOrchestrator.cs

```diff
@@ -761,10 +761,11 @@ private CommitmentListItemViewModel MapFrom(CommitmentView listItem, string late
             };
         }
 
-        private async Task<List<ITrainingProgramme>> GetTrainingProgrammes()
+        private async Task<List<ITrainingProgramme>> GetTrainingProgrammes(bool includeFrameworks)
```

Comment:
Small point (same as in employercommitments), I'd have supplied a default value to default to existing behaviour. Also, an enum would make calling code easier to read than a bool flag.